history_test.go

```diff
@@ -16,28 +17,57 @@ type testWriteCloser struct {
 func (wrc *testWriteCloser) Close() error { return nil }
 
 func TestExecute(t *testing.T) {
-	command := "echo testing"
-	var output bytes.Buffer
-	r, err := history.NewRecorder()
-	if err != nil {
-		t.Fatal(err)
-	}
-	err = r.EnsureHistoryFileOpen()
-	if err != nil {
-		fmt.Fprint(r.Stdout, err)
-	}
-	r.Stdout = &output
-	err = r.Execute(command)
-	if err != nil {
-		t.Fatal(err)
+	testCases := []struct {
+		command     string
+		desc        string
+		errExpected bool
+		output      bytes.Buffer
```

Comment:
I don't see where `output` is set in any of the test cases?

Reply:
line 51: r.Stdout = &tC.output